Gemfile

```diff
@@ -1,5 +1,4 @@
 source 'https://rubygems.org'
 
-gem 'jekyll',     '=0.11.0'
-gem 'liquid',     '=2.2.2'
-gem 'RedCloth',   '=4.2.9'
+gem 'github-pages'
+gem 'kramdown'
```

Comment:
Any particular reason for adding the kramdown gem here? It's included in the github-pages gem.